### PR TITLE
Added description key to tag details

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19754

## Description
Added description key to tag details

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
@kirbles19 
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/a0645d472c1ca827a0ae4c00ea7d64db0058b953/Integrations/AutofocusV2/CHANGELOG.md)

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@kirbles19

**Comment by ShahafBenYakir** [comment]
@yuvalbenshalom Added specific verification to the new enhancement

File: Integrations/AutofocusV2/AutofocusV2.py [hunk 1]
```diff
@@ -805,7 +806,7 @@ def search_samples_command():
     first_seen = argToList(args.get('first_seen'))
     last_updated = argToList(args.get('last_updated'))
     query = args.get('query')
-    scope = args.get('scope')
+    scope = args.get('scope').capitalize()
```
Comment (ShahafBenYakir) [inline Integrations/AutofocusV2/AutofocusV2.py]: @yuvalbenshalom This should solve the issue we discussed earlier

**Comment by kirbles19** [comment]
@ShahafBenYakir Done, and docs are updated wtih the new output.

**Comment by ShahafBenYakir** [comment]
@kirbles19 Thank you

**Pull request merged**